tdrs-backend/tdpservice/email/helpers/test/test_account_deactivation_warnings.py

```diff
@@ -0,0 +1,20 @@
+"""Test functions for email helper."""
+import pytest
+
+import tdpservice
+from tdpservice.email.helpers.account_deactivation_warning import send_deactivation_warning_email
+
+
+@pytest.mark.django_db
+def test_send_deactivation_warning_email(mocker, user):
+    """Test that the send_deactivation_warning_email function runs."""
+    mocker.patch(
+        'tdpservice.email.helpers.account_deactivation_warning.automated_email.delay',
+        return_value=None
+    )
+    send_deactivation_warning_email([user], 10)
+
+    assert tdpservice.email.email.automated_email.delay.called_once_with(
+        recipient_email=user.email,
+        subject='Account Deactivation Warning: 10 days remaining',
+    )
```

Comment:
For this unit test would it be worth it to mock a date and assert that the `deactivation_date` logic works? I can be swayed either way for the reason of why you also wouldn't test third party libs

Reply:
Test like that are in another file. Test covers the functionality of send_deactivation_warning_email.